services/provider/client/client.go

```diff
@@ -31,10 +31,10 @@ func NewGRPCConnection(serverAddr string, opts []grpc.DialOption) (*grpc.ClientC
 
 // OnBoardConsumer to validate the consumer and create StorageConsumer
 // resource on the StorageProvider cluster
-func (cc *ConsumerClient) OnBoardConsumer(token, capacity string) (*pb.OnBoardConsumerResponse, error) {
+func (cc *ConsumerClient) OnBoardConsumer(name, capacity string) (*pb.OnBoardConsumerResponse, error) {
```

Comment:
`OnBoardConsumer` -> `OnboardConsumer`

Onboard is a single word (according to google search)

Reply:
using `onboard` and `offboard`